src/Arcus.Observability.Telemetry.Serilog.Enrichers/KubernetesEnricher.cs

```diff
@@ -8,6 +8,9 @@ namespace Arcus.Observability.Telemetry.Serilog.Enrichers
     /// <summary>
     /// Enrichment on log events that automatically adds Kubernetes information from the environment.
     /// </summary>
+    /// <remarks>
```

Comment:
Added the following remark as I implemented it incorrectly the first I implemented it.